web/src/installers/installer-versions.ts

```diff
@@ -41,3 +68,25 @@ export async function getInstallerVersions(distUrl: string, kurlVersion?: string
   installerVersionsCache[url] = installerVersions;
   return installerVersionsCache[url];
 }
+
+async function externalAddonHandler() {
+  try {
+    const response = await fetch("https://kurl-sh.s3.amazonaws.com/external/addon-registry.json");
+    externalAddons = await response.json();
+  } catch (error) {
+    logger.error(error, "failed to pull external addon registry.");
```

Comment:
Just making sure here, we're intending for this function to be a best-effort and just log an error?

Reply:
Yeah, I'm not 100% sure what to do here.  If the file isn't available that's reallllly bad but if the application already has data cached, it could already be current and i wouldn't want to fail a request because of that.